### DISCUSSION TITLE
Not getting "granules"

### DISCUSSION THREAD

**Discussion by gpb3037** [earthlab-education/Earth-Analytics-AY24#581] [Q&A]
in the redlining assignment I see the geographic bounds being ok for Denver
denver_redlining_gdf.total_bounds
array([-105.06225,   39.62952, -104.87626,   39.79111])

but when I run the code to open data - it never says anything about granules regardless of what I specify including
denver_results = earthaccess.search_data(
    short_name="HLSL30",
    bounding_box=tuple(denver_redlining_gdf.total_bounds),
    temporal=("2023-07-17"), # works but no granules 
    # gives error temporal=("2023-07-17","2023-07-18"),
    # temporal=("2023-07-17"),
    # works temporal=("2023-07-12","2023-07-13"),
    # temporal=("2023-07-01","2023-07-31"),
    count=1
    
)
denver_results
and I get results like this which say there is a data size, etc
[Collection: {'EntryTitle': 'HLS Landsat Operational Land Imager Surface Reflectance and TOA Brightness Daily Global 30m v2.0'}
 Spatial coverage: {'HorizontalSpatialDomain': {'Geometry': {'GPolygons': [{'Boundary': {'Points': [{'Longitude': -104.87922543, 'Latitude': 39.66154416}, {'Longitude': -104.54781482, 'Latitude': 40.64997117}, {'Longitude': -105.00023656, 'Latitude': 40.65085652}, {'Longitude': -105.00023316, 'Latitude': 39.66160696}, {'Longitude': -104.87922543, 'Latitude': 39.66154416}]}}]}}}
 Temporal coverage: {'RangeDateTime': {'BeginningDateTime': '2023-07-19T17:43:04.956Z', 'EndingDateTime': '2023-07-19T17:43:04.956Z'}}
 Size(MB): 49.664061546325684
 Data: ['https://data.lpdaac.ea

but I get the error AttributeError: 'NoneType' object has no attribute 'open'
when I try
denver_file_data = earthaccess.open(denver_results)

this is at about the 24 minute mark of the video for part 2 data download

**Comment by eculler** [comment]
For anyone else with this problem -- @gpb3037 and I determined that this was a problem with incorrect login information. Unfortunately that manifests as this confusing error at a later stage, instead of stating the login failed!

**Comment by hanried** [comment]
I was also running into this, thank you for asking @gpb3037 and thanks for answering @eculler!

**Comment by allenmoench** [comment]
@eculler @gpb3037 what was the fix for this - is there a .reset command or something like that I should use? I'm doing the bulk download and having the same issue where I don't get a granule, but it doesn't ask me for my login info even when I restart the kernel. Thanks! ~ Mike

**Reply by gpb3037** [reply]
My process was to go back through the video step by step using the initial template as at some point I must have tweaked something.   Consider following exactly the dates selected in the video.  The other thing is that there is no report back if your username or password is not accepted, and the generic password I had been using had to be tweaked when I was using the data site to something I did not have as generic (but I forgot this and repeatedly tried my generic password).  A third thing to look at would be the date of your request - I thought mine was cloudless on the data site but somehow I was not seeing the correct date there and got a cloudy day.
